types/ramda/index.d.ts

```diff
@@ -1209,18 +1209,18 @@ export function mapAccumRight<T, U, TResult>(fn: (acc: U, value: T) => [U, TResu
 type PartialRecord<K extends keyof any, T> = {
     [P in K]?: T;
 };
-export function mapObjIndexed<T, TResult, TKey extends string>(
+export function mapObjIndexed<T, TResult, TKey extends string | number | symbol>(
```

Comment:
You can use `keyof any` as in `PartialRecord`, which provides the same union (`string | number | symbol`), but more semantic and future proof

Reply:
> You can use `keyof any` as in `PartialRecord`, which provides the same union (`string | number | symbol`), but more semantic and future proof

Please feel free to PR this.